C++/MST_Kruskals.cpp

```diff
@@ -0,0 +1,79 @@
+#include<bits/stdc++.h>

```

Comment:
Add some comments to both files to explain the logics

Reply:
Okay @kaustubhgupta, thanks for the review 👍 
I will do that